third_party/move/move-vm/runtime/src/loader/mod.rs

```diff
@@ -429,38 +437,41 @@ pub(crate) struct LoaderV1 {
     // other transactions.
     module_cache_hits: RwLock<BTreeSet<ModuleId>>,
 
-    vm_config: VMConfig,
+    // The environment used by the loader, e.g, VM config, available native functions, etc..
+    runtime_env: Arc<RuntimeEnvironment>,
```

Comment:
nit: let's call runtime_environment

Reply:
done, update here, in function names, and in other files